jarbas/chamber_of_deputies/tests/test_reimbursement_view.py

```diff
@@ -1,8 +1,13 @@
+import sys
+
```

Comment:
I see no need for this blank line separating `sys` from other native Python module imports. What do you think about it?